src/usr/local/www/services_ntpd.php

```diff
@@ -335,7 +338,7 @@ function build_interface_list() {
 	'Max candidate pool peers',
 	'number',
 	$pconfig['ntpmaxpeers'],
-	['min' => 4, 'max' => $max_candidate_peers]
+	['min' => $min_candidate_peers, 'max' => $max_candidate_peers]
 ))->setHelp('Maximum candidate NTP pool peers. Remember that many servers inside pools are provided by volunteers, ' .
 	'all you will gain from a higher number is extra load on the volunteer time servers. (Default: 5).');
 
```

Comment:
Reword to avoid addressing the user directly ("you").

Perhaps: 
> Maximum number of candidate peers in the NTP pool. This value should be set low enough to provide sufficient alternate sources while not contacting an excessively large number of peers. Many servers inside public pools are provided by volunteers, and a large candidate pool places unnecessary extra load on the volunteer time servers for little to no added benefit. (Default: 5).